src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -752,19 +799,37 @@ def create_cc_resource(client, resource_group_name, cluster_name, cc, no_wait):
         utils.arm_exception_handler(e, consts.Create_ConnectedCluster_Fault_Type, 'Unable to create connected cluster resource')
 
 
+def patch_cc_resource(client, resource_group_name, cluster_name, cc, no_wait):
```

Comment:
>patch_cc_resource [](http://example.com/codeflow?start=4&length=17)

Club both of these methods to a one and switch based on a flag.

Reply:
done